extensions/amp-story/1.0/amp-story-vertical.css

```diff
@@ -14,13 +14,12 @@
  * limitations under the License.
  */
 
-amp-story[standalone][scroll] {
-  overflow-y: auto !important;
+amp-story[standalone][vertical] {
   height: auto !important;
   contain: initial !important;
 }
 
-[scroll] amp-story-page {
+[vertical] amp-story-page {
```

Comment:
Can we add a comment here so that future readers know what the vertical attribute is for?

Reply:
Done